src/models.py

```diff
@@ -44,10 +59,14 @@ def check_if_both_age_and_country(self) -> Self:
 
 class SuperUser(BaseModel, validate_assignment=True):
     user_id: PositiveInt
-    username: Annotated[str, StringConstraints(min_length=2, pattern=r"^[a-zA-Z]*$")]
+    username: str = Field(
+        min_length=2,
+        pattern=r"^[a-zA-Z]*$",
+        description="Username must contain only letters and be at least two characters long.",
+    )
     email: EmailStr
-    age: Optional[PositiveInt]
-    country: Optional[CountryAlpha2]
+    age: PositiveInt | None
+    country: CountryAlpha2 | None
     superuser_id: PositiveInt
     role: SuperUserRoles
```

Comment:
I see that SuperUser now isn't inherited from User model anymore, was there a practical reason?

Reply:
Reverted to inheritance